Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -3119,12 +3119,28 @@ void CGContextDrawTiledImage(CGContextRef context, CGRect rect, CGImageRef image
 #pragma endregion
 
 #pragma region Drawing Operations - Gradient + Shading
+
+static bool _isFloatEqual(const float a, const float b) {
+    static float epsilon = 1e-7;
+    return fabsf(a - b) < epsilon;
```

Comment:
already being used here in context & it's from math.h

---
In reply to: [107499794](https://github.com/Microsoft/WinObjC/pull/2249#discussion_r107499794) [](ancestors = 107499794)